src/main/java/io/prometheus/cloudwatch/WebServer.java

```diff
@@ -33,11 +33,10 @@ public static void main(String[] args) throws Exception {
         server.setHandler(context);
         context.addServlet(new ServletHolder(new MetricsServlet()), "/metrics");
         context.addServlet(new ServletHolder(new DynamicReloadServlet(collector)), "/-/reload");
-        context.addServlet(new ServletHolder(new HealthServlet()), "/-/healthy");
-        context.addServlet(new ServletHolder(new HealthServlet()), "/-/ready");
+        context.addServlet(new ServletHolder(new HealthServlet(collector)), "/-/healthy");
+        context.addServlet(new ServletHolder(new HealthServlet(collector)), "/-/ready");
```

Comment:
For the use case I'm trying to address, I certainly need the `/-/healthy` endpoint to return a non-200 response code if there is a scrape error. I'm not sure if `/-/ready` should have the same behavior since in Kubernetes, readiness simply indicates whether the endpoints for a pod should be revealed to the service that targets the pod. Since the `/metrics` endpoint will still be up even if there is a scrape error, do you think I should the configuration of the `/-/ready` endpoint as it was?